integration-testing/pom.xml

```diff
@@ -54,9 +54,24 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-simple</artifactId>
-            <version>1.6.3</version>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-lambda-java-log4j2</artifactId>
```

Comment:
Shouldn't need lambbda logging here - integration tests don't do any aws stuff - just make http requests to a deployed stack.